TM1py/Services/CellService.py

```diff
@@ -477,25 +537,15 @@ def extract_cellset_raw(
         elif 'Ordinal' not in cell_properties:
             cell_properties.append('Ordinal')
 
-        if not elem_properties:
-            elem_properties = ['UniqueName']
-        elif 'UniqueName' not in elem_properties:
-            elem_properties.append('UniqueName')
-
-        if not member_properties:
-            member_properties = ['UniqueName']
-        elif 'UniqueName' not in member_properties:
-            elem_properties.append('UniqueName')
```

Comment:
seems to be an old bug, but in line 488 it should say:
member_properties.append('UniqueName')